### PR TITLE
{devel}[GCCcore/9.3.0] PySide2 v5.14.1, Shiboken2 v5.14.1 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by SimonPinches**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@SimonPinches could you take a look at the issues raised by the tests please?

**Comment by SimonPinches** [comment]
@verdurin I've added `bintutils` build dependencies as mentioned in the test issues but I might need some clues if more is needed...

File: easybuild/easyconfigs/p/PySide2/PySide2-5.14.1-GCCcore-9.3.0-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = ['https://download.qt.io/official_releases/QtForPython/%(namelower)s']
+sources = ['%(name)s-%(version)s-%(version)s-cp35.cp36.cp37.cp38-abi3-manylinux1_x86_64.whl']
+checksums = ['5b0cf7f0a9cb8fb11881b8615c7d979f5d02bdab6c3f428bf40956537c87071c']
+
+use_pip = True
+unpack_sources = False
+download_dep_fail = True
+
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/PySide2/PySide2-5.14.1-GCCcore-9.3.0-Python-3.8.2.eb:20]: need
sanity_pip_check = True
here

File: easybuild/easyconfigs/s/Shiboken2/Shiboken2-5.14.1-GCCcore-9.3.0-Python-3.8.2.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = ['https://download.qt.io/official_releases/QtForPython/%(namelower)s']
+sources = ['%(namelower)s-%(version)s-%(version)s-cp35.cp36.cp37.cp38-abi3-manylinux1_x86_64.whl']
+checksums = ['b046f810f2409b734621972ee6bff1eac76e87f08ab0638010d99a1b79f72842']
+
+use_pip = True
+unpack_sources = False
+download_dep_fail = True
+
```
Comment (akesandgren) [inline easybuild/easyconfigs/s/Shiboken2/Shiboken2-5.14.1-GCCcore-9.3.0-Python-3.8.2.eb:20]: need
sanity_pip_check = True
here

**Review by akesandgren — CHANGES_REQUESTED** [review]

**Comment by trz42** [comment]
Closing this PR, since it (only) uses deprecated toolchain versions (GCC/9.3.0),
see https://docs.easybuild.io/policies/toolchains .

Also ...
- If a new PR for this software with a supported toolchain is opened, the minor comments about sanity checks should be taken into account.

**Comment by boegel** [comment]
Also worth mentioning: a newer version of `PySide2` was merged in https://github.com/easybuilders/easybuild-easyconfigs/pull/13460

**Pull request closed without merging**